plugin/path_static_account.go

```diff
@@ -0,0 +1,346 @@
+package gcpsecrets
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/errwrap"
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+const (
+	staticAccountStoragePrefix       = "static"
+	staticAccountPathPrefix          = "static"
+	gcpServiceAccountInferredProject = "-"
+)
+
+func pathStaticAccount(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: fmt.Sprintf("%s/%s", staticAccountPathPrefix, framework.GenericNameRegex("name")),
+		Fields: map[string]*framework.FieldSchema{
```

Comment:
Might be good to mention which fields cannot be updated after creation in their `Description`.

Reply:
Added in a7e9ea70ce064a17ad5d14db0e192beaab44f825